project/scripts/winCmdTests.bat

```diff
@@ -2,7 +2,7 @@
 setlocal
 
 @rem paths are relative to the root project directory
-set "_PREFIX=dist\target\pack"
+set "_PREFIX=dist\win-x86_64\target\pack"
```

Comment:
I understand that the native version is tested here `project/scripts/native-integration/winTests.bat`. We should probably revert this change and keep testing with the `jar` version

Reply:
but the whole reason we do the native runners is because the windows runner only has jvm 8 on it so the test breaks